assignment-2/7s.md

```diff
@@ -1,2 +1,120 @@
-#7S
-// TODO
\ No newline at end of file
+#7S Framework.
+
+> Assignment - 7S:
+> Organisations are not just about methodology. There are quite a number of abstractions for organisations. We will look at the 7S model. Now we go and look outside to two successful companies through the eyes of the 7S model. You can select these companies. One should be a software company, one should come from another industry. What can you learn from this for your failed case? What can you learn from this about the methods RUP and Scrum.
+> 
+> Look for organisations that really make clear how they work from the inside. Think about Etsy, Quora, Google, Facebook, Apple, Microsoft, Mozilla, Amazon, Spotify.
+
+## 1. Introduction.
```

Comment:
I do not understand why exactly give a 'Wikipedia'-like explanation of 7S. This is not asked in the assignment. We can start the chapter with an introduction, but then it should be about what we exactly investigated (which companies, why, how, methods etc.). The current introduction can be removed, if you ask me.